.node-version

```diff
@@ -1 +1 @@
-20.8.1
+20.9.0
```

Comment:
Per #299, let's hold off on the Node 20.9 upgrade until CircleCI and Docker support it